src/logic/scripting/lua/libs/libworld.cpp

```diff
@@ -115,6 +119,76 @@ static int l_get_generator(lua::State* L) {
     return lua::pushstring(L, require_world_info().generator);
 }
 
+static int l_get_chunk_data(lua::State* L) {
+    int x = (int)lua::tointeger(L, 1);
+    int y = (int)lua::tointeger(L, 2);
+    const auto& chunk = level->chunks->getChunk(x, y);
+    if (chunk->isEmpty()) {

```

Comment:
potential nullptr dereference

Reply:
Yes, i noticed that isEmpty is a Chunk method, not sharedptr